src/index.js

```diff
@@ -71,6 +72,10 @@ export default function (debug = printDebug) {
       })
     }
 
+    function buildAssertionsOutOfWrapperObject(wrapperObject) {
+      Object.keys(wrapperObject).forEach(key => overwriteMethod(wrapperObject[key], key))
```

Comment:
The project is already setup to use babel and ES6, I didn't want to add a extra dependency for something I could solve natively.